test/extensions/filters/http/cache/simple_http_cache/simple_http_cache_test.cc

```diff
@@ -257,6 +276,28 @@ TEST_F(SimpleHttpCacheTest, VaryResponses) {
   EXPECT_TRUE(expectLookupSuccessWithBody(first_value_vary.get(), Body1));
 }
 
+TEST_F(SimpleHttpCacheTest, PutGetWithTrailers) {
+  const std::string RequestPath1("Name");
```

Comment:
I know the rest of the tests have request path variables like this, and so you are staying consistent with the file, but we should provide a real path in the `RequestPath#` vars, rather than just an arbitrary string, ie. with a leading slash: `"/name"`. This has no functional impact right now, but it will make a difference as we add more features in the near future.

The rest of the file is wrong, and is something we should fix at some point, while simultaneously not adding more with the same mistake.

Reply:
also I don't think we use the RequestPath1 spelling for local consts. The Envoy style guide is not specific but it points to the Google style guide which says:

> and whose value is fixed for the duration of the program

So I think just name this `request_path1`.

Reply:
Hi! Thank you for your comments! Would be happy to update the path, but I am not sure to what value. So, we could leave it like this for now or if you give me a little more context on what it should include(there does not seem to be consensus in references), I could update it.

Reply:
Suggest:
```
const std::string request_path1 = "/path";
```